hr_job_category/models/hr.py

```diff
@@ -42,13 +58,31 @@ def create(self, vals):
         return res
 
     def write(self, vals):
-        if "employee_id" in vals:
-            self.mapped("employee_id").write({"category_ids": [(5,)]})
+        prev_data = self.read(["job_id"])
```

Comment:
prev_data could be a dict:
```suggestion
        prev_data = {
            res["id"]: res["job_id"][0]
            for res in self.read(["job_id"]) if res["job_id"]
        }
```